### ISSUE TITLE
OmniSharp doesn't recognize mscorlib namespaces for a buildable project

### ISSUE THREAD

**Issue by firelizzard18** [OmniSharp/omnisharp-vscode#2005]
## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (2.1.3)

Product Information:
 Version:            2.1.3
 Commit SHA-1 hash:  a0ca411ca5

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.13
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /usr/local/share/dotnet/sdk/2.1.3/

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.4
  Build    : 7f262f453d8c8479b9af91d34c013b3aa05bc1ff
```

- VSCode Version: Code 1.19.3 (7c4205b5c6e52a53b81c69d2b2dc8a627abaa0ba, 2018-01-25T10:25:01.248Z)
- OS Version: Darwin x64 17.4.0
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
csharp|ms-|1.13.1

## Steps to reproduce
Have a project. Build it successfully with `dotnet build`. Open it in VSCode.

## Expected  behavior
OmniSharp does its job.

## Actual behavior
Lots of errors like this: `The type or namespace name 'System' could not be found (are you missing a using directive or an assembly reference?)`

**Comment by firelizzard18** [comment]
This seems to mostly happen when I'm looking at files inside submodules.

**Comment by DustinCampbell** [comment]
@firelizzard18: Could you provide your OmniSharp Log after loading your project? You can locate this in VS Code by selecting View->Output and then choosing "OmniSharp Log" from the top-right corner of the Output pane.

**Comment by jasoncavett** [comment]
I'm seeing this as well. I have 5000+ errors in my project (which builds and runs otherwise) with errors like, `The type or namespace name 'System' could not be found (are you missing a using directive or an assembly reference?)`

```
dotnet --info
.NET Command Line Tools (2.1.4)

Product Information:
 Version:            2.1.4
 Commit SHA-1 hash:  5e8add2190

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.16299
 OS Platform: Windows
 RID:         win10-x64

 Base Path:   C:\Program Files\dotnet\sdk\2.1.4\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.5
  Build    : 17373eb129b3b05aa18ece963f8795d65ef8ea54
```

Some additional details.

Running ms-vscode.csharp version 1.14.0 (which, interesting enough, is marked "Preview" though I don't remember choosing that).

In the problems list, I see it referencing .NETCoreApp,Version=v1.1.AssemblyAttribute.cs in `C:\Users\Jason Cavett\AppData\Local\Temp`

See attachment for OmniSharp log.
[omnisharp-log.txt](https://github.com/OmniSharp/omnisharp-vscode/files/1728656/omnisharp-log.txt)

**Comment by barcexwiki** [comment]
I have the same problem. I had to roll back from 1.14.0 to 1.13.1 to get it fixed.

**Comment by firelizzard18** [comment]
I haven't had a chance to try to reproduce this issue, but I could believe I'm having a similar issue to @barcexwiki, as I do use `<TargetFrameworks>` instead of `<TargetFramework>` in some situations. Unfortunately for me, I must use the plural form in those situations.

**Comment by cramhead** [comment]
I was having the same problem. I have an asp.net core web app with targeted to netcoreapp2.0 and it refers to a library targeted to netstandard2.0. I tried changing the library to netcoreapp2.0 but it could not restore. I reverted the library change and tried setting the web app to netstandard2.0. As one might expect that failed too. When i reverted the web app change, so that everything was exactly the same as when I started I no long had the errors. 
I would have to expect that it's problem with cached data that was somehow reset by my flailing.

```
dotnet --info
.NET Command Line Tools (2.1.4)

Product Information:
 Version:            2.1.4
 Commit SHA-1 hash:  5e8add2190

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.13
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /usr/local/share/dotnet/sdk/2.1.4/

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.5
  Build    : 17373eb129b3b05aa18ece963f8795d65ef8ea54
```
Similarly, I'm running ms-vscode.csharp version 1.14.0 (which, interesting enough, is marked "Preview" though I don't remember choosing that).
